### ISSUE TITLE
Make "Open DevTools in Web Browser" a top-level command

### ISSUE THREAD

**Issue by f-person** [Dart-Code/Dart-Code#5000]
**Is your feature request related to a problem? Please describe.**
I only use DevTools in the web browser, and having to go through an additional menu for that becomes annoying over time :)

**Describe the solution you'd like**
It'd be great to have the "Open DevTools in Web Browser" command be accessible on a top-level through ⌘+⇧+P (default shortcut)

**Describe alternatives you've considered**
-

**Additional context**
I think it also makes sense to have it since we have all the other ones as top-level AFAIK

**Comment by DanTup** [comment]
Have you tried setting the `dart.devToolsLocation` setting?

```js
"dart.devToolsLocation": {
	"default": "external"
},
```

This will cause DevTools to always open externally. That way you can just use the button on the toolbar for Inspector, or any of the existing DevTools commands that take you directly to the page you want.

**Comment by f-person** [comment]
Oh, thanks! This is a great workaround; I still occasionally use the inspector in Code itself, though, so it'd be nice to eventually have this :)

**Comment by DanTup** [comment]
@f-person I don't know if it helps, but you can set a different option for each section:

```js
"dart.devToolsLocation": {
	"inspector": "beside",
	"default": "external"
}
```

**Comment by f-person** [comment]
Yeah, I noticed this and did something similar already, but didn't know I could just specify the default; that's great, thanks! I do think it'd still be nice to add it, though :)
